### PR TITLE
Fix compiler warning

### LINKED ISSUE #6233: Compile warning: unused `_keyManager`
**Describe the bug**
```
> Compilation warnings encountered:

    /home/unlock/smart-contracts/contracts/mixins/MixinKeys.sol:271:5: Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
    address _keyManager
    ^-----------------^
```

**To Reproduce**
yarn build

### PR DESCRIPTION
**Pull request by nfurfaro**
# Description
FIxes the use of params inside the `isKeyManager` internal function.

# Issues

Fixes #6233 
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: smart-contracts/contracts/mixins/MixinKeys.sol [hunk 1]
```diff
@@ -272,8 +272,8 @@ contract MixinKeys is
   ) internal view
     returns (bool)
   {
-    if(keyManagerOf[_tokenId] == msg.sender ||
```
Comment (julien51) [inline smart-contracts/contracts/mixins/MixinKeys.sol]: Should there be a pair of tests which make sure it passes with `_keyManager` and fails with `msg.sender` in the case where the sender is not the `_keyManager`?

Comment (nfurfaro) [reply smart-contracts/contracts/mixins/MixinKeys.sol]: I could add this. This function is not tested directly yet.

Comment (nfurfaro) [reply smart-contracts/contracts/mixins/MixinKeys.sol]: added some tests.

File: smart-contracts/contracts/mixins/MixinKeys.sol [hunk 2]
```diff
@@ -266,14 +266,14 @@ contract MixinKeys is
   * Returns true if _keyManager is the manager of the key
   * identified by _tokenId
    */
-  function isKeyManager(
+  function _isKeyManager(
     uint _tokenId,
     address _keyManager
   ) internal view
```
Comment (NickCuso) [inline smart-contracts/contracts/mixins/MixinKeys.sol]: Should we make this public instead?

Comment (nfurfaro) [reply smart-contracts/contracts/mixins/MixinKeys.sol]: I don't know. We already have the public mapping `keyManagerOf` available. This seems like something more needed internally maybe?

**Comment by nfurfaro** [comment]
Yes, a good review of our permissions and maybe more testing before v7.

File: smart-contracts/contracts/mixins/MixinKeys.sol (hunk 2 above)
Comment (nfurfaro) [reply smart-contracts/contracts/mixins/MixinKeys.sol]: We can always make this public if the need arises. Will merge as-is for now.

**Pull request merged**